src/js/labels/type.ts

```diff
@@ -0,0 +1,17 @@
+/* Smaller label object for representing labels in a list */
+export type LabelNested = {
+    /* A hex encoded color */
+    color: string;
+    /* A detailed description */
+    description: string;
+    /* A unique identifier */
+    id: number;
+    /* A name for the label */
+    name: string;
+};
+
+/* A label object that extends LabelNested */
```

Comment:
```suggestion
/* A sample label */
```
`Label` is the default case so there isn't much to say here.